lock-api/src/main/java/com/palantir/lock/HeldLocksToken.java

```diff
@@ -48,12 +52,17 @@
     /**
      * This should only be created by the Lock Service
      */
-    public HeldLocksToken(BigInteger tokenId, LockClient client, long createionDateMs,
-            long expirationDateMs, SortedLockCollection<LockDescriptor> lockMap,
-            TimeDuration lockTimeout, @Nullable Long versionId) {
-        this.tokenId = Preconditions.checkNotNull(tokenId);
-        this.client = Preconditions.checkNotNull(client);
-        this.creationDateMs = createionDateMs;
+    public HeldLocksToken(
```

Comment:
yeah, this is more just from a stylistic consistency PoV, and more broadly making sure API can ser/deser appropriately (i.e. why didn't we this elsewhere earlier).